### ISSUE TITLE
Editor: Page scrolls to the top when saving

### ISSUE THREAD

**Issue by jasmussen** [Automattic/wp-calypso#8971]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/post/
2. Write a long post, so long that it needs to be scrolled
3. Scroll to the bottom
4. Press the save button
#### What I expected

I expected the viewport not to scroll to the top, making me lose my place in the text. 
#### What happened instead

The viewport scrolled me to the top, perhaps to show me the "Post updated" notice.
#### Screenshot / Video

Here's a video showing saving, then jumping to the top:

https://cloudup.com/cBp5IRH2uMa
#### Context / Source

After publishing I like to edit > update > edit > update, and losing my place every time makes for a jumpy experience. 

Instead of scrolling to the top to show the notice, the existing UI affordance for save status could let the user know a post has been saved:

<img width="254" alt="screen shot 2016-10-27 at 11 35 43" src="https://cloud.githubusercontent.com/assets/1204802/19762238/8a1b3d0a-9c39-11e6-81a1-5d2c27123011.png">

**Comment by Tagada85** [comment]
I would to try working on this issue!

**Comment by Tagada85** [comment]
Ok, let me give a quick update of where I am right now. I believe I might need some guidance to finish this.
Here is how I partly solved the problem.

The reason why the page scrolls back to the top was the presence of `window.scrollTo(0,0)` in **post-editor/post-editor.jsx** when we call the _onSaveSuccess()_ method. So I removed this, and the page doesn't scroll back to the top.

Now, obviously, I needed to make sure that the page was still showing the notice. So, I added a few styles to **components/notice/styles.scss** : 

- position: fixed, 
- z-index: 100 (So the notice is in front of the blog post)

I also added a margin-top of 50px to the **post-editor/editor-title/styles.scss** to make sure the notice wasn't partly in front of the title.

My problem is this: the width of the notice. I can't figure out how to make sure the notice width changes appropriately on different screen sizes.

The notice does show at the top of the screen and do not scroll back on save. But I 'just' need to figure out the width issue.

I would love to know what you think!
